README.md

```diff
@@ -372,9 +371,3 @@ ks delete ${KF_ENV} -c ambassador
 ## CONTRIBUTING
 
 Please feel free to test the system! [developer-guide.md](./docs/developer-guide.md) is a good starting point for developers.
-
-## TODOs
-
-* Integrate KubeFlow (TensorFlow, Caffe2 and PyTorch operators)
-* Support Early Stopping
-* Enrich the GUI
```

Comment:
these TODOs  aren't still done

Reply:
This is README though. Should they be tracked somewhere else, e.g. roadmap, issues, story board, etc.?

Reply:
OK, maybe you can create a doc to trace it

Reply:
I think we could use issues to keep track of the TODO items. WDYT

Reply:
make sense